metadata-ingestion/tests/integration/powerbi/test_m_parser.py

```diff
@@ -736,3 +738,26 @@ def test_sqlglot_parser():
         data_platform_tables[1].urn
         == "urn:li:dataset:(urn:li:dataPlatform:snowflake,sales_deployment.operations_analytics.transformed_prod.v_sme_unit_targets,PROD)"
     )
+
+    expected_columns: List[str] = [
+        "client_director",
+        "tier",
+        'upper("manager")',
+        "team_type",
+        "date_target",
+        "monthid",
+        "target_team",
+        "seller_email",
+        "agent_key",
+        "sme_quota",
+        "revenue_quota",
+        "service_quota",
+        "bl_target",
+        "software_quota",
+    ]
+
+    actual_columns: List[str] = [
+        column_lineage.downstream.column for column_lineage in lineage[0].column_lineage
```

Comment:
for this test, we should check the full `lineage[0].column_lineage` object

e.g. it should be 

```
assert lineage[0].column_lineage = [
	ColumnLineageInfo(downstream=something, upstreams=[something1, something2]),
	ColumnLineageInfo(...),
	...
]
```

Reply:
done